assets/js/script.js

```diff
@@ -226,6 +239,94 @@ function displayCovidStatsSecondary(country) {
 
 };
 
+function searchHistory(country1, country1Value, country2, country2Value) {
+
+    // If no second country selected
+    if (!country2) {
+        var countryObj = {"name": country1, "id0": country1Value};
+    } else {
+        var countryObj = {"name": country1 + " & " + country2, "id0": country1Value, "id1": country2Value};
+    }
+
+    // remove duplicate countryObj in historyArr
+    removeDuplicates(countryObj);
+
+    // unshift new countryObj to front of historyArr
+    historyArr.unshift(countryObj);
+
+    // set localStorage
+    // localStorage.setItem("covid-country-search-items", historyArr);
+    
+    // clear search btns
+    document.getElementById("search-history-buttons").innerHTML = "";
+
+    // create search history btns
+    for (var i = 0; i < historyArr.length; i++) {
+        var searchHistoryBtn = document.createElement("button");
+        searchHistoryBtn.textContent = historyArr[i].name;
+        searchHistoryBtn.className = "search-history-btn one-third";
```

Comment:
@gmmarcoux can you add CSS styling in your feature branch to these search history buttons?